streams/co.m3u

```diff
@@ -109,7 +109,27 @@ https://canalzoom.itex.com.co:3027/live/canalzoombr1live.m3u8
 https://movil.ejeserver.com/live/zuldemayda.m3u8
 #EXTINF:-1 tvg-id="ZuldemaydaCanal13.co" status="timeout",Zuldemayda Canal 13 (Armenia | Quindio)
 https://video.ejeserver.com/live/zuldemayda.m3u8
-#EXTINF:-1 tvg-id="" status="blocked",Canal Caracol (720p) [Geo-blocked]
+#EXTINF:-1 tvg-id="CaracolTV.co" status="blocked",Caracol TV (720p) [Geo-blocked]
 https://mdstrm.com/live-stream-playlist/574463697b9817cf0886fc17.m3u8
-#EXTINF:-1 tvg-id="" status="timeout",Ondambiantal TV (Bogotà | Cundinamarca)
+#EXTINF:-1 tvg-id="OndabientalTV.co" status="timeout",Ondambiantal TV (Bogotà | Cundinamarca)
 https://server12.videostreaming.net:3230/stream/play.m3u8
+#EXTINF:-1 tvg-id="Canal12.co",Canal 12 (Valledupar) (486p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://teveo.com.co/player/AcAAAgAJAAFFA1IAyADIQCAAAAAAAAAA.js
+#EXTINF:-1 tvg-id="CanalCartagena.co",Canal Cartagena [Not 24/7]
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://teveo.com.co/player/AeAAAgAlAANrA1IAyADIQCAAAAAAAAAAA.js
+#EXTINF:-1 tvg-id="ChamberofRepresentatives.co",Cámara de Representantes [Not 24/7]
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCzescUFTh2jiZkYIx9uZtqA/live
+#EXTINF:-1 tvg-id="CNCMonteria.co",CNC Monteria (720p)
+https://cloud28.vsgtech.co:3932/live/cncmonterialive.m3u8
+#EXTINF:-1 tvg-id="Cosmovision.co",Cosmovision (720p)
+https://videohls2.cosmovision.tv/hls/sd.m3u8
+#EXTINF:-1 tvg-id="FamiliChannel.co",Famili Channel (720p)
+https://cp.panelchs.com:1936/8012/8012/playlist.m3u8
+#EXTINF:-1 tvg-id="PSCTelevision.co",PSC Televisión
+https://cloud28.vsgtech.co:3052/live/psctvlive.m3u8
+#EXTINF:-1 tvg-id="TVPenol.co",TV Peñol (720p)
+https://stmv1.voxtvhd.com.br/tvpenol/tvpenol/playlist.m3u8
+#EXTINF:-1 tvg-id="RCNMas.co",RCN Más (1080p)
+https://rcntv-rcnmas-1-us.plex.wurl.tv/playlist.m3u8
```

Comment:
This one should be placed on us_plex.m3u. We also have a roku variant available.